glue/cirq/stimcirq/_cirq_to_stim_test.py

```diff
@@ -339,6 +339,21 @@ def test_on_loop():
     result = stimcirq.StimSampler().run(c)
     assert result.measurements.keys() == {'0:a', '0:b', '1:a', '1:b', '2:a', '2:b'}
 
+def test_loop_with_one_rep():
+    q0 = cirq.q(0)
+    cc = cirq.Circuit(
+        cirq.CircuitOperation(
+        cirq.FrozenCircuit(
+            cirq.Moment(cirq.H(q0)),
+            cirq.Moment(cirq.H(q0)),
+            cirq.Moment(cirq.H(q0)),
+            cirq.Moment(cirq.H(q0)),
+        )
+        )
+    )
+    sc = stimcirq.cirq_circuit_to_stim_circuit(cc)
+    assert str(sc).splitlines().count('TICK') > 1
```

Comment:
Assert the actual circuit's contents instead of asserting something abstract about it.

Reply:
Made the change!